src/commands/install.rs

```diff
@@ -75,16 +75,50 @@ pub fn install(
     }
 
     // Update local manifest
+    // -------------------------------------------------------------------------
```

Comment:
*Nit:* I'd prefer to not have line comments that add sections like this. If the code isn't clear without them, then we probably should refactor it; and if it is clear enough without them, then they aren't needed.